### PR TITLE
Create ULLafayette_Hei.yaml

### PR DESCRIPTION
**Pull request by aowen-uwmad**
Per https://support.opensciencegrid.org/a/tickets/75652

### CODE REVIEW

**Comment by mwestphall** [comment]
@opensciencegrid/project-office do we have permission to create a new institution for "University of Louisiana at Lafayette"? There is an existing institution for "Louisiana State University".

**Comment by jstathas** [comment]
Approved. Can you also add this into the DB since I believe my access is down with the rework?

Janet


From: mwestphall ***@***.***>
Date: Thursday, April 18, 2024 at 11:31 AM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] Create ULLafayette_Hei.yaml (PR #3811)

@opensciencegrid/project-office<https://github.com/orgs/opensciencegrid/teams/project-office> do we have permission to create a new institution for "University of Louisiana at Lafayette"? There is an existing institution for "Louisiana State University".

**Review by mwestphall — APPROVED** [review]

File: projects/ULLafayette_Hei.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Department: Center for Advanced Computer Studies
+Description: While fine-tuning large language models (LLMs) to follow instructions
```
Comment (mwestphall) [inline projects/ULLafayette_Hei.yaml:2]:
```suggestion
Description: >
While fine-tuning large language models (LLMs) to follow instructions
```

File: projects/ULLafayette_Hei.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Department: Center for Advanced Computer Studies
+Description: >
+While fine-tuning large language models (LLMs) to follow instructions
```
Comment (mwestphall) [inline projects/ULLafayette_Hei.yaml:3]:
```suggestion
  While fine-tuning large language models (LLMs) to follow instructions
```

File: projects/ULLafayette_Hei.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Description: >
+  While fine-tuning large language models (LLMs) to follow instructions
+  has emerged as a pivotal paradigm, enabling promising applications such as ChatGPT
+  and GitHub Copilot, the capability to follow arbitrary instructions also introduces
+  acute vulnerabilities. This project aims to deepen understanding of the security
+  and privacy risks associated with large instruction-tuned language models and to
+  obtain insights on how to mitigate these risks.
+FieldOfScience: Computer and Information Science and Engineering
+Organization: University of Louisiana at Lafayette
+PIName: Xiali Hei
```
Comment (mwestphall) [inline projects/ULLafayette_Hei.yaml:11]:
```suggestion
PIName: Xiali Hei
InstitutionID: 'https://osg-htc.org/iid/2cp294m0uq0z'
```

**Pull request merged**